supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -8,6 +8,21 @@ Follow the _IBM Style_ guide advice of _day Month year_ for date formats, for ex
 
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
+[[man-pages]]
+== Man page references
+
+When referencing a man page in an additional resources section, use the following format: `_<man_page_name>(<section_number>)_` _man page on your system_.
+
+Do not link to a website containing the man page information. The contents of a man page may vary between systems or package versions, so users must run the `man` command on their system to view the most accurate information for the named command or utility.
```

Comment:
```suggestion
Do not link to a website that contains the man page information. 
The contents of a man page might vary between systems or package versions, so users must run the `man` command on the target system to view the system-specific information for the named command or utility.
```

Reply:
Thanks! I used your wording suggestions, but kept them on the same line (because the info about users having to run the `man` command on their system is a reason for why we say not to link to man page websites)

Reply:
Putting them on separate lines was just to make it easier to review (each sentence on a separate line) -- it didn't affect the output, the two sentences were still in the same paragraph. But it was a tiny suggestion, no problem at all for not implementing it :)

Reply:
Gotcha - yeah I normally don't write my sentences that way like I know some people do :)